tests/models/whisper/test_modeling_whisper.py

```diff
@@ -88,7 +88,7 @@ def __init__(
         seq_length=60,
         is_training=True,
         use_labels=False,
-        vocab_size=99,
+        vocab_size=51865,
```

Comment:
The whisper tokenizer has special tokens with the high ids, so we need to keep the vocabulary to avoid index out of bound errors.

Reply:
This breaks equivalence test with TF, the numbers are still close, but above the threshold.

Reply:
Normally OOV should always be decoded as `""` in Whisper.
`tokenizer.decode(50364)` sould give `""`

Reply:
I got this one, 99 is too little because we have more special tokens than that, but 300 should be enough. The index out of bound error was because of the hardcoded token ids in the logits processor:

https://github.com/huggingface/transformers/blob/91ff7efeeb3e6bb10d83702db24108bb6583e013/src/transformers/generation/logits_process.py#L936-L937

I changed it such that we pass those ids when building the processor.